mythril/laser/ethereum/svm.py

```diff
@@ -132,10 +148,13 @@ def sym_exec(
 
         if main_address:
             log.info("Starting message call transaction to {}".format(main_address))
+            self.set_standard_initial_state(self.world_state.accounts)
```

Comment:
this can be moved before the if statement, so we don't need the statement on ln 157 as well